src/System.Text.Json/ref/Configurations.props

```diff
@@ -1,11 +1,12 @@
 ﻿<Project DefaultTargets="Build">
   <PropertyGroup>
+    <PackageConfigurations>
+      netstandard;
+    </PackageConfigurations>
     <BuildConfigurations>
+      $(PackageConfigurations)
       netcoreapp;
```

Comment:
FYI: @ahsonkhan looks like we do have a ref assembly built specifically for netcoreapp and UAP which is exposing the DisposeAsync method. This creates the problems that we discussed offline with @ericstj . I don't plan to fix that on this PR, but we should definitelly create a new issue to track that and remove these extra configurations if possible.